main/webapp/modules/core/scripts/project/menu-bar-extensions.js

```diff
@@ -0,0 +1,519 @@
+		if(services[i].url){
+			ids.push(services[i].url);
+		}
+	}
+	/*
+	$.post("command/rdf-extension/initializeServices",{"services":JSON.stringify(ids)},function(data){
+		RdfReconciliationManager.registerService(data);
+		if(onDone){
+			onDone();
+		}
+	},"json");
+	*/
+};
+
+
+//extend the column header menu
+$(function(){
+    
+	ExtensionBar.MenuItems.push(
+			{
+				"id":"reconcile",
+				"label": "RDF",
+				"submenu" : [
+					{
+						"id": "rdf/edit-rdf-schema",
+						label: "Edit Semantic Model...",
+						click: function() { RdfExporterMenuBar.editRdfSchema(false); }
+					},
+					{
+						"id": "rdf/reset-rdf-schema",
+						label: "Reset Semantic Model...",
+						click: function() { RdfExporterMenuBar.editRdfSchema(true); }
+					},
+					{},
+			        {
+			        	"id": "rdf/reconcile",
+			            label: "Add reconciliation service",
+			            submenu:[
+			                     {
+			                    	 "id" :"rdf/reconcile/sparql",
+			                    	 label: "Based on SPARQL endpoint...",
+			                    	 click: function() { RdfReconciliationManager.newSparqlService(); }
+			                     },
+			                     {
+			                    	 "id":"rdf/reconcile/dump",
+			                    	 label:"Based on RDF file...",
+			                    	 click: function() { RdfReconciliationManager.newRdfService(); }        	 
+			                     },
+			                     // {
+			                    	//  "id" : "rdf/reconcile/sindice",
+			                    	//  label: "Based on a Sindice site search...",
+			                    	//  click: function() { RdfReconciliationManager.newSindiceService(); }        	 
+			                     // },
+//			                     {
+//			                    	 "id" : "rdf/reconcile/stanbol",
+//			                    	 label: "Based on a Apache Stanbol EntityHub...",
+//			                    	 click: function() { RdfReconciliationManager.newStanbolService(); }        	 
+//			                     }
+						]
+					
+			        }
+			    ]
+			}
+	);
+	DataTableColumnHeaderUI.extendMenu(function(column, columnHeaderUI, menu) {
+		MenuSystem.appendTo(menu, [ "core/reconcile" ], [
+		                                             {},
+	//	                                             {
+	//	                                                 id: "core/sindice-find-dataset",
+	//	                                                 label: "Discover related RDF datasets..." ,
+	//	                                                 click: function() {
+	//	                                                     var dialog = new SindiceDialog();
+	//	                                                     dialog.show(column);
+	//	                                                 }
+	//	                                             },
+		                                         ]);
+	});
+	
+	
+	RdfReconciliationManager.synchronizeServices();
+});
+
```

Comment:
Why is this javascript file marked as new?

Reply:
This is overwritten by the ant script build.xml